assets/src/scripts/monitoring-location/components/hydrograph/date-controls.js

```diff
@@ -102,20 +133,22 @@ export const drawDateRangeControls = function(elem, store, siteno) {
         .attr('id', 'custom-date-submit')
         .text('Submit')
         .on('click', function() {
-            const userSpecifiedStart = customStartDate.node().value;
-            const userSpecifiedEnd = customEndDate.node().value;
+            let userSpecifiedStart = document.getElementById('custom-start-date').value
```

Comment:
I know this breaks the d3 convention but it was the only way to get it to work with my trials. There were some issues accessing the value property using d3 methods.